BE/src/Validation/UserValidationRules/userUpdate.ts

```diff
@@ -1,12 +1,13 @@
 import { body } from "express-validator";
 
 export const updateUserValidationRules = [
-    body("permissions")
-        .custom((value) => {
-            if (!Array.isArray(value)) {
-                throw new Error("Permissions must be an array.");
-            }
-            return true;
-        })
-        .withMessage("permissions should be an array."),
+    body("id")
+        .exists()
+        .withMessage("id must exist to update user.")
+        .isString()
+        .withMessage("id must be a string.")
+        .isLength({ min: 6 })
```

Comment:
![image](https://github.com/user-attachments/assets/51643da5-257c-4a4d-9b1b-116f90bf8759)

You may also want to help the user with their password to make sire they understand what restrictions there are